LEAF_Request_Portal/form.php

```diff
@@ -3036,10 +3036,13 @@ public function query($inQuery)
 
         if($joinRecordResolutionData)
         {
+            require_once 'VAMC_Directory.php';
+            $dir = new VAMC_Directory;
+
             $conditions .= 'records_workflow_state.stepID IS NULL AND submitted > 0 AND deleted = 0 AND ';
             $joins .= 'LEFT JOIN records_workflow_state USING (recordID) ';
 
-            $res2 = $this->db->prepared_query('SELECT recordID, lastStatus, records_step_fulfillment.stepID, fulfillmentTime FROM records
+            $res2 = $this->db->prepared_query('SELECT recordID, userID as resolvedBy, lastStatus, records_step_fulfillment.stepID, fulfillmentTime FROM records
```

Comment:
This userID that you are pulling from this query is not the resolvedBy individual its the person who started the request.